docs/compass/02-03-runtime-agent-workflow.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Runtime Agent architecture
+type: Architecture
+---
+
+This document presents the workflow of Runtime Agent. 
+
+![Runtime Agent architecture](./assets/runtime-agent-architecture.svg)
+
+1. Runtime Agent fetches the certificate from the Connector to initialize connection with Compass.
+
+2. Runtime Agent stores the certificate and key for the Connector and the Director in the Secret.
+
+3. Runtime Agent synchronizes the Runtime with the Director by fetching new Applications from the Director and creating them in the Runtime, and removing from the Runtime the Applications that no longer exist in the Director. 
```

Comment:
```suggestion
3. Runtime Agent synchronizes the Runtime with the Director by fetching new Applications from the Director and creating them in the Runtime. It also removes from the Runtime the Applications that no longer exist in the Director. 
```

Reply:
It's one sentence 'cause it's also part of this synchronization mentioned at the beginning.

Reply:
Can we rephrase and divide it into two shorter sentences in any other way? Such long sentences are a bit confusing and for sure are to be avoided in technical docs. How about introducing bullet points? Something like this:
```
3. Runtime Agent synchronizes the Runtime with the Director by:
- fetching new Applications from the Director and creating them in the Runtime
- removing from the Runtime the Applications that no longer exist in the Director
```

What do you think?